source/transfer_list.rst

```diff
@@ -413,6 +413,35 @@ Inputs:
 
    #. `void_te.size` = `te.data_size + te.hdr_size - 0x8`
 
+Overwriting a TE
+^^^^^^^^^^^^^^^^
+
+Inputs:
+
+- `tl_base_addr`: Base address of the TL from which TE to be overwritten
```

Comment:
nit: I'd argue tl_base_addr is not an argument to the Overwrite TE operation.
The tl_base_addr is used in the "Adding a new TE" procedure, which is executed if the "Overwriting a TE" aborts.